### ISSUE TITLE
Clone fork in the original owner's directory (via option?)

### ISSUE THREAD

**Issue by yu-ichiro** [siketyan/ghr#132]
Currently `clone --fork` will clone the forked repository as it is, but making `clone --fork` clone the fork into the original owner's directory might be more useful.

```
current: ~/.ghr/github.com/yu-ichiro/ghr
proposal: ~/.ghr/github.com/siketyan/ghr (cloned from yu-ichiro/ghr)
```

This has some advantages such as:
* being able to distinguish the forked repository by its original owner
* `ghr browse` will open the original repo (which IMO used more often)

Btw adding upstream to remotes after cloning a fork will be very useful and I was thinking it might be good to release these features together

**Comment by siketyan** [comment]
Yes, I also want this. But I prefer configuring it by the config file (ghr.toml) instead of the command-line option. I think this dicision is partially permanent for a user.
